src/components/SeedsSelectedList/index.js

```diff
@@ -8,7 +8,40 @@ import {
   CardActionArea,
 } from '@mui/material';
 import { useDispatch, useSelector } from 'react-redux';
-import { selectSidebarSeedRedux } from '../../features/calculatorSlice/actions';
+import {
```

Comment:
I really like this icon. 2 things. 
I think we should either add this to our shared items so that we can use this x repeatedly 
or
we need to use an MUI icon that is provided to us to make sure there wont be any issues with compatibility later.  
@YazidZidane  what do you think?

Reply:
Yeah we could build this to a shared component, I'm not sure if there's a MUI icon like this but if there is, we could also use that for simplicity.